src/protocol.cpp

```diff
@@ -2114,11 +2115,19 @@ bool CProtocol::EvaluateCLChannelLevelListMes  ( const CHostAddress&     InetAdd
 }
 
 void CProtocol::CreateCLRegisterServerResp  ( const CHostAddress& InetAddr,
-                                              const int           iStatus )
+                                              const ESvrRegStatus eStatus )
 {
     int              iPos = 0; // init position pointer
     CVector<uint8_t> vecData( 1 );
 
+    int iStatus = static_cast<int> ( eStatus );
+    if ( iStatus < ESvrRegStatus::SRS_MIN_RESP
+         || iStatus > ESvrRegStatus::SRS_MAX_RESP )
+    {
+        qWarning() << "CProtocol::CreateCLRegisterServerResp: eStatus" << eStatus << "invalid value";
```

Comment:
please remove the qWarning()

Reply:
There doesn't appear to be any way to tell the caller of the create method that it's been called badly.  Have got any ideas what could be done in that case?

Reply:
Hmm, is it not just for debugging during developement? Is it intended to give a warning for the end user of Jamulus (i.e. not a code developer)?

Reply:
Actually, as this is only _called_ by the "central server" code, it should never trigger (if the range of values was increased, the MIN and MAX would increase - all this would do would highlight that).  So yes, it's actually useless.  I'll get rid of it.